task_manager/task_manager.cc

```diff
@@ -0,0 +1,44 @@
+#include "task_manager.hh"
+
+task_manager::tasks_per_module& task_manager::get_tasks() {
+    return _tasks;
+}
+
+seastar::shared_ptr<task_manager::task> task_manager::get_task(utils::UUID id) {
```

Comment:
When https://github.com/scylladb/scylladb/pull/11220 is merged this should use `using task_id = utils::tagged_uuid<struct task_id_tag>;`